### DISCUSSION TITLE
Let's remove `api` from the `io.netty5.buffer.api` package name

### DISCUSSION THREAD

**Discussion by chrisvest** [netty/netty#12789] [Ideas]
The `api` part is from before we differentiated the Netty 5 packages with `io.netty5`, so it was used to keep the old and new APIs separate.

However, now that the Netty 5 packages are differentiated from Netty 4, and we've removed ByteBuf from Netty 5, there's no longer a need to keep this extra level. It looks out of place.

On the other hand, changing it is going to impact a lot of code, including all of the new contrib repositories.

Thoughts?

**Comment by normanmaurer** [comment]
Now is the time

**Comment by chrisvest** [comment]
Let's goo! https://github.com/netty/netty/pull/12792
